pkgs/development/tools/pnpm/fetch-deps/default.nix

```diff
@@ -74,6 +80,7 @@
                 --force \
                 --ignore-scripts \
                 ${installFlags} \
```

Comment:
Should be removed? Or Perhaps kept above?

Reply:
It is removed. Not sure where you're seeing that.

Reply:
> It is removed. Not sure where you're seeing that.

I reviewed the PR commit by commit, and this `${installFlags}` usage was removed only in a later commit. It is a nit pick, so I wouldn't have commented about this if this would have been my only comment.